web-api/src/personalization/routes.rs

```diff
@@ -169,14 +172,7 @@ async fn personalized_documents(
     .map(|documents| {
         if let Some(documents) = documents {
             Either::Left(Json(PersonalizedDocumentsResponse {
-                documents: documents
-                    .into_iter()
-                    .map(|document| PersonalizedDocumentData {
-                        id: document.id,
-                        score: document.score,
-                        properties: document.properties,
-                    })
-                    .collect(),
+                documents: documents.into_iter().map(Into::into).collect(),
```

Comment:
:art: `Itertools` provides a shorthand for that
```suggestion
                documents: documents.into_iter().map_into().collect(),
```

Reply:
I will change it but I'm not the biggest fan of it because `map(Into::into)` works on all monade like types but `map_into` only works on iterators.